src/apify/_configuration.py

```diff
@@ -296,7 +304,7 @@ class Configuration(CrawleeConfiguration):
                 'actor_web_server_url',
                 'apify_container_url',
             ),
-            description='Unique public URL for accessing the Actor run web server from the outside world',
+            description='Unique public URL for accessing a specific Actor run web server from the outside world.',
```

Comment:
All the other descriptions don't have a period at the end:
```suggestion
            description='Unique public URL for accessing a specific Actor run web server from the outside world',
```